awx/main/isolated/manager.py

```diff
@@ -324,24 +324,28 @@ def health_check(self, instance_qs):
                         task_result = runner_obj.get_fact_cache(instance.hostname)
                     except Exception:
                         logger.exception('Failed to read status from isolated instances')
-                    if 'awx_capacity_cpu' in task_result and 'awx_capacity_mem' in task_result:
```

Comment:
Instead of this, what if you just filtered at the top of this method e.g.,

```python
instance_qs = instance_qs.filter(enabled=True)
```

Reply:
Makes sense, updated